StantUiIosLibrary/Classes/Tables/ContactsTableViewWithCollapsedSearchBar/ContactsTableViewWithCollapsedSearchBar.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ContactsTableViewWithCollapseSearchBar.swift
+//  StantUiIosLibrary
+//
+//  Created by Renato Vieira on 27/06/19.
+//
+
+import UIKit
+
+public class ContactsTableViewWithCollapseSearchBar: UIView {
+
+    public var tableView:         ContactsTableView?
+    public var searchView:        DefaultSearchBar?
+    public var emptyMessageLabel: UILabel?
+    
+    weak var tableViewDelegate:   ContactsTableViewDidSelectDelegate?
+    
+    var searchBarIcon        = UIImage()
+    var searchBarPlaceholder = String()
+    var searchOnTableView    = String()
+    var emptyMessage         = String()
+    public var constructionSiteList = [ContactsInformation]()
+    
+    var currentSearch = ""
+    
+    required public init?(coder aDecoder: NSCoder) {
+           super.init(coder: aDecoder)
+    }
+       
+    public override init(frame: CGRect) {
+           super.init(frame: frame)
+    }
+    
+    public func configureViewWith(constructionList:     [ContactsInformation],
+                                  searchBarIcon:        UIImage,
+                                  searchBarPlaceholder: String,
+                                  tableViewDelegate:    ContactsTableViewDidSelectDelegate,
+                                  emptyMessage:         String) {
+        
+        self.tableViewDelegate    = tableViewDelegate
+        self.constructionSiteList = constructionList
+        self.searchBarIcon        = searchBarIcon
+        self.searchBarPlaceholder = searchBarPlaceholder
+        self.emptyMessage         = emptyMessage
+        self.backgroundColor      = .backgroundStant
+        
+        self.configureEmptyMessageLabel()
+        self.configureSearchView()
+        self.configureTableView()
+        self.anchorSearchAndTableView()
+        self.setEmptyMessageLabelVisibility()
+    }
+    
+    fileprivate func configureSearchView() {
+        searchView = DefaultSearchBar(frame: CGRect(x: 0, y: 0, width: self.frame.width, height: DefaultSearchBar.searchViewHeight))
+        
+        guard let searchView = searchView else { return }
+        self.addSubview(searchView)
+        searchView.configureViewWith(delegate:        self,
+                                     image:           searchBarIcon,
+                                     placeholderText: searchBarPlaceholder)
+    }
+    
+    fileprivate func configureTableView() {
+        tableView = ContactsTableView(frame: CGRect(x: 0, y: DefaultSearchBar.searchViewHeight, width: self.frame.width, height: self.frame.height))
```

Comment:
Break and indent the parameters on the lines 65, 66 and 67.

Reply:
Done